_data/simple-icons.json

```diff
@@ -14066,6 +14066,12 @@
             "source": "https://design.sencha.com",
             "guidelines": "https://design.sencha.com/productlogo.html"
         },
+        {
+            "title": "SendGrid",
+            "hex": "55A8DD",
```

Comment:
```suggestion
            "hex": "51A9E3",
```
This is the hex listed under "SendGrid Colors" for "SendGrid Blue", and the color I'm finding in the SVGs provided at the download page. Where did you find the other one?

Reply:
There's a colour version of the icon on their brand resources page: https://sendgrid.com/en-us/resource/brand

Reply:
> There's a colour version of the icon on their brand resources page: [sendgrid.com/en-us/resource/brand](https://sendgrid.com/en-us/resource/brand).

The color in the SVG available for download is the same as the color listed too.

Reply:
Ah, you are using the EPS file. There is an SVG available for download if you click the "Download Brand Assets" button:

![CleanShot 2024-10-10 at 10 24 25](https://github.com/user-attachments/assets/bea8afef-ba38-45ad-b039-fe5473765c6f)